api/apis/job_handler.go

```diff
@@ -34,26 +36,27 @@ func (h *JobHandler) jobGetHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	jobGUID := vars["guid"]
 
-	spaceGUID := getSpaceGUID(jobGUID)
-	if spaceGUID == "" {
+	var jobResponse presenter.JobResponse
+	if strings.HasPrefix(jobGUID, syncSpacePrefix) {
+		spaceGUID := strings.Replace(jobGUID, syncSpacePrefix, "", 1)
+		jobResponse = presenter.ForManifestApplyJob(jobGUID, spaceGUID, h.serverURL)
+	} else if strings.HasPrefix(jobGUID, appDeletePrefix) {
+		jobResponse = presenter.ForAppDeleteJob(jobGUID, h.serverURL)
+	} else {
 		h.logger.Info("Invalid Job GUID")
 		writeNotFoundErrorResponse(w, "Job")
 		return
 	}
 
-	err := writeJsonResponse(w, presenter.ForJob(jobGUID, spaceGUID, h.serverURL), http.StatusOK)
+	responseBody, err := json.Marshal(jobResponse)
 	if err != nil {
 		h.logger.Error(err, "Failed to render response", "Job GUID", jobGUID)
 		writeUnknownErrorResponse(w)
+		return
 	}
-}
 
-func getSpaceGUID(jobGUID string) string {
-	if strings.HasPrefix(jobGUID, syncSpacePrefix) {
-		spaceGUID := strings.Replace(jobGUID, syncSpacePrefix, "", 1)
-		return spaceGUID
-	}
-	return ""
+	w.WriteHeader(http.StatusOK)
```

Comment:
I think we normally omit this as this is the default status code.

Reply:
Good call I think I missed this when rebasing on main along with some other artifacts of letting the branch languish over the weekend. Will remove.